cli/installer/templates/config.yaml.tpl

```diff
@@ -0,0 +1,32 @@
+postgres:
+  host: {{ .pHost }}
+  user: {{ .pUser }}
+  password: {{ .pPasswd }}
+  port: 5432
+  dbname: postgres
+  params: sslmode=disable
+
+poolingConfig:
+  maxWaitTimeForTrace: 2m
+  retryDelay: 3s
+
+googleAnalytics:
+  enabled: {{ .analyticsEnabled }}
+
+demo:
+  enabled: [{{ .enabledDemos }}]
+  endpoints:
+    pokeshopHttp: {{ .pokeshopHttp }}
+    pokeshopGrpc: {{ .pokeshopGrpc }}
+    otelFrontend: {{ .otelFrontend }}
+    otelProductCatalog: {{ .otelProductCatalog }}
+    otelCart: {{ .otelCart }}
+    otelCheckout: {{ .otelCheckout }}
+
+experimentalFeatures: []
+
+{{ if eq .installBackend "true" }}
+server:
+  telemetry:
+    dataStore: {{ .backendType }}
```

Comment:
Do we need this line in the config file?

Reply:
I don't think we need it because we have the provision file

Reply:
You are right folks! I've removed this part from the config file.